### PR TITLE
Feature/tribe v2

### PR DESCRIPTION
**Pull request by sanchaymittal**
##### Description

1. Gitcoin staff can edit header view of tribes
- With this tribe header can be edited by gitcoin staff now.

2. Ordered User profile & Townsquare side-bar UI
- Allows to fetch correct ordering of user profile based on rank_coder
- sidebar UI elements will now show name only.
- before it was showing followers count only.

##### Refers/Fixes

#7310
#7295

##### Testing
https://www.loom.com/share/56038d60ecf34d3aadf6e3980c2f5fd3

### CODE REVIEW

File: app/assets/v2/js/vue-components.js [hunk 1]
```diff
@@ -701,28 +700,23 @@ Vue.component('suggested-profile', {
...
+<b-media tag="li" class="row mx-auto mx-md-n1 mb-1">
   <template v-slot:aside>
     <a :href="profile_url" class="d-flex nav-link nav-line pr-0 mr-0">
       <b-img :src="avatar_url" class="nav_avatar"></b-img>
     </a>
   </template>
-  <div class="row">
-    <span class="col-6 col-md-12 col-xl-7 font-caption">
+  <div class="col">
+    <span class="row-6 row-md-12 row-xl-7 font-caption">
```
Comment (thelostone-mc) [inline app/assets/v2/js/vue-components.js:710]: Does this work ?

File: app/assets/v2/js/users.js [hunk 2]
```diff
@@ -9,7 +9,7 @@ let hackathonId = document.hasOwnProperty('hackathon_id') ? document.hackathon_i
 Vue.mixin({
   computed: {
     orderedUsers: function() {
-      return _.orderBy(this.users, 'position_contributor', 'asc');
+      return _.orderBy(this.users, 'rank_coder', 'desc');
```
Comment (thelostone-mc) [inline app/assets/v2/js/users.js:12]: The ordering should happen on the backend !
Do remove the `_` library and other changes as well which were introduced

**Review by thelostone-mc — CHANGES_REQUESTED** [review]
left a few comments

File: app/assets/v2/js/vue-components.js (hunk 1 above)
Comment (sanchaymittal) [reply app/assets/v2/js/vue-components.js:710]: Yesss

Comment (sanchaymittal) [reply app/assets/v2/js/vue-components.js:710]: change

cc: @PixelantDesign

Comment (octavioamu) [reply app/assets/v2/js/vue-components.js:710]: what exactly are those row- classes?

Comment (octavioamu) [reply app/assets/v2/js/vue-components.js:710]: '.row' is a bootstrap grid classe to hold cols, cols are responsive but doesn't make sense to me have row responsive as is not what control sizes.

Comment (octavioamu) [reply app/assets/v2/js/vue-components.js:710]: you are using 4.5 booastrap doc, if you notice those row- doesn't exist on the bootstrap version we are using 
https://getbootstrap.com/docs/4.3/layout/grid/ 
If you use the inspector will notice those doesn't do nothing

File: app/assets/v2/js/users.js (hunk 2 above)
Comment (sanchaymittal) [reply app/assets/v2/js/users.js:12]: removing it.

File: app/assets/v2/js/vue-components.js (hunk 1 above)
Comment (sanchaymittal) [reply app/assets/v2/js/vue-components.js:710]: Ohh right! I will make the changes based on v4.3 and push it.

**Review by thelostone-mc — APPROVED** [review]

**Review by octavioamu — APPROVED** [review]

**Pull request merged**